src/hooks/useFeature.ts

```diff
@@ -0,0 +1,24 @@
+import { useQuery } from 'react-query';
+import { useActiveAccount } from './useActiveAccount';
+import { useHttpClient } from './useHttpClient';
+
+type Response<T extends string> = Record<T, boolean>;
+
+export function useFeature<T extends string>(feature: T) {
+  const { accountHeaders } = useActiveAccount();
+  const { httpClient } = useHttpClient();
+
+  return useQuery(
+    ['Features', feature],
+    () =>
+      httpClient.get<Response<T>>(`/v1/features/${feature}`, {
+        headers: accountHeaders,
+      }),
+    {
+      enabled: !!accountHeaders,
+      select(data) {
```

Comment:
Oo, nice - dig the usage of this method instead of `.then` in the `queryFn`, thereby keeping the original `data` intact in cache.  We should use this more often / throughout - I'm guilty of a lot of `.then`s in here.